packages/api-reference-editor/test/main.ts

```diff
@@ -0,0 +1,42 @@
+import { mountApiReferenceEditable } from '@/api-reference-editor'
+
+const { updateSpecValue } = mountApiReferenceEditable(
+  '#scalar-api-reference-editor',
+  { useExternalState: true },
+  (v: string) => {
+    updateSpecValue(v)
+    console.log('input')
+  },
+)
+
+setTimeout(
+  () =>
+    updateSpecValue(`openapi: 3.0.3
```

Comment:
Maybe we can use the new Scalar galaxy example here, usage examples here:

https://github.com/scalar/scalar/pull/1539 (was merged just now)